understanding/20/images-of-text-no-exception.html

```diff
@@ -9,6 +9,15 @@
    <h1>Understanding Images of Text (No Exception)</h1>
    
    
+   <section id="brief">
+      <h2>In brief</h2>
+      <dl>
+         <dt>Objective</dt><dd>Only use text that can be adapted by users</dd>
```

Comment:
This feels a bit off compared to the others - this objective is a command for authors to follow, rather than a description of the desired user-facing goal. Maybe instead something like this?

```suggestion
         <dt>Objective</dt><dd>Users can control how text is presented to them</dd>
```

(maybe "can always control" here for AAA and "can control" for the AA)

Reply:
Just got down to "Visual Presentation", which uses "Text appearance can be altered by users to meet preferences"  - that would also be fine to reuse

Reply:
I'm going to spend a bit more time trying to think of another way of expressing this. There's a real jump from the SC title to the actual objective. Maybe I can find a softer description that conveys the gist...

Reply:
How about this, @dbjorge?
```suggestion
         <dt>Objective</dt><dd>Users can adjust how text is presented</dd>
```